policy/predicate/file.go

```diff
@@ -32,24 +32,55 @@ type ChangedFiles struct {
 
 var _ Predicate = &ChangedFiles{}
 
-func (pred *ChangedFiles) Evaluate(ctx context.Context, prctx pull.Context) (bool, string, error) {
+func (pred *ChangedFiles) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
 	files, err := prctx.ChangedFiles()
```

Comment:
This should come after you create the `PredicateResult` so that it is immediately followed by the `if` statement checking the error.